package.json

```diff
@@ -188,40 +203,28 @@
     "node-loader": "2.0.0",
     "npm-run-all": "4.1.5",
     "playwright": "1.28.1",
+    "pretty-bytes": "6.0.0",
     "ps-node": "^0.1.6",
+    "react": "16.14.0",
+    "react-beautiful-dnd": "13.1.0",
+    "react-bootstrap": "1.6.4",
+    "react-dom": "16.14.0",
+    "react-intl": "5.20.10",
+    "react-select": "5.2.2",
     "recursive-readdir": "^2.2.2",
     "robotjs": "0.6.0",
+    "sass": "1.49.11",
     "sass-loader": "12.6.0",
+    "semver": "7.3.5",
     "shebang-loader": "0.0.1",
     "style-loader": "3.3.1",
     "ts-prune": "0.10.3",
     "typescript": "4.6.3",
+    "valid-url": "1.0.9",
     "webpack": "5.71.0",
     "webpack-cli": "4.10.0",
     "webpack-dev-server": "4.8.0",
     "webpack-merge": "5.8.0",
-    "@mattermost/compass-icons": "0.1.32",
```

Comment:
Since this is a new feature (and could be reverted if something goes wrong) can we split this fix into a separate PR?

Reply:
Oh I'm okay with this, I just wanted to separate it out into a separate PR, since it's more of a cleanup step that's not in scope.